docs/book/stacks-and-components/component-guide/orchestrators/sagemaker.md

```diff
@@ -119,6 +133,27 @@ side. From there you can view the pipelines that have been launched via ZenML:
 
 ![Sagemaker Studio Pipelines](../../../.gitbook/assets/sagemakerUI.png)
 
+### Debugging SageMaker Pipelines
+
+If your SageMaker pipeline failed before the first ZenML step was started, the
+ZenML run will not show up in the ZenML dashboard. In these cases, you need to 
+[use the SageMaker UI](#sagemaker-ui) to view the respective error message and
+logs. To do so, open the corresponding pipeline in the SageMaker UI, then open
```

Comment:
A lot of assumption that the person will know how to open the pipeline in the Sagemaker UI. Maybe just a bit more information / hints about where they can go to find this?

Reply:
That's shown in a lot of detail incl. screenshots in the previous section (#sagemaker-ui)